crates/biome_js_formatter/src/context/trailing_comma.rs

```diff
@@ -90,9 +88,9 @@ impl FromStr for TrailingComma {
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         match s {
-            "es5" | "ES5" => Ok(Self::Es5),
-            "all" | "All" => Ok(Self::All),
-            "none" | "None" => Ok(Self::None),
```

Comment:
Another breaking change we can do as part of a refactor. Those values must be deprecated first